homeassistant/components/justnimbus/strings.json

```diff
@@ -15,5 +15,45 @@
     "abort": {
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "pump_flow": {
+        "name": "Pump flow"
+      },
+      "drink_flow": {
+        "name": "Drink flow"
+      },
+      "pump_pressure": {
+        "name": "Pump pressure"
+      },
+      "pump_starts": {
+        "name": "Pump starts"
+      },
+      "pump_hours": {
+        "name": "Pump hours"
+      },
+      "reservoir_temperature": {
+        "name": "Reservoir temperature"
```

Comment:
It shouldn't matter, but this was previously `Reservoir Temperature` but now the capitalization has changed to `Reservoir temperature`

Reply:
Yes, we like all entities to use title case unless it's a brand name of some sorts